src/utils/database-common/src/entities.rs

```diff
@@ -46,6 +46,17 @@ pub struct EntityPageListing<Entity> {
     pub total_count: usize,
 }
 
+// Entity may be a type that doesn't implement Default, so we implement it
+// manually
+impl<Entity> Default for EntityPageListing<Entity> {
```

Comment:
Why does it matter? Entity construction is not involved when we create an empty Vec